com.github.miguelmota.Cointop.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Cointop
+Exec=com.github.miguelmota.Cointop
```

Comment:
So close, this needs to be the name of the binary. (Also please do test this by installing it and opening it from your launcher)

Reply:
@TingPing would `Exec=cointop` be correct? I'm testing it on ubuntu-server so I don't have a launcher